js/mainjs/addIPCListeners.js

```diff
@@ -1,56 +1,53 @@
-'use strict';
-
 // Electron main process libraries
-const Electron = require('electron');
-const Dialog = Electron.dialog;
-const IPCMain = Electron.ipcMain;
-const Menu = Electron.Menu;
+import { dialog, ipcMain, Menu } from 'electron'
```

Comment:
ES6 member imports (`import { blah ...}`) import exported members directly by name.  As such, they must match the names defined in the module.